roles/include_components/tasks/main.yml

```diff
@@ -1,14 +1,14 @@
 ---
 - name: Track development git repos
   ansible.builtin.include_tasks: track_dev_git_repo.yml
-  loop: "{{ ic_dev_gits }}"
```

Comment:
Is there any reason for changing the name of these variables from `ic_*` to `include_components_*`? I believe we agreed on defining the role-related variables by using a prefix with the initial letters of the role name, in lower case.

Also, if this change is done, it will also affect dci-openshift-agent and dci-openshift-app-agent, since they're using these variables too.

Reply:
Okey, just answering myself, because it's happening to me in https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/114: it's like the Ansible lint check requires to use the role name as a prefix for the variables passed to the role, it's no longer valid to use the abbreviation. Then, I don't know if we'll have to correct all the roles to work in this way :sweat_smile: 

If that's correct, as I said, the agents need to be changed too

Reply:
Where do you see the errors you describe?

For this change the summary https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/7401686151?pr=113 shows the problem is a yaml file that can't be loaded due to a syntax error

For #114 the issue is different, the issue is a very long line, also shown in the summary: https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/7408665692?pr=114

Reply:
Regarding

> Is there any reason for changing the name of these variables from ic_* to include_components_*? I believe we agreed on defining the role-related variables by using a prefix with the initial letters of the role name, in lower case.

I agree with @ramperher, while ansible-lint recommends using full role name, let's not make those changes in here, we can discuss how and when switch to that standard

Reply:
@tonyskapunk , the problem is that, at least in my PR, I was not able to merge because of that ansible-lint recommendation. See this check, which was run after fixing my issue with the line that was exceeding the recommended length: https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/7408600528

```
-ERROR: roles/add_day2_node/tasks/main.yml:23: yaml[truthy]: Truthy value should be one of
-ERROR: roles/add_day2_node/tasks/main.yml:34: yaml[truthy]: Truthy value should be one of
-ERROR: roles/add_day2_node/tasks/main.yml:44: var-naming[no-role-prefix]: Variables names from within roles should use patch_host_config_ as a prefix. (vars: discovered_host)
-ERROR: roles/add_day2_node/tasks/main.yml:44: var-naming[no-role-prefix]: Variables names from within roles should use patch_host_config_ as a prefix. (vars: inventory_nodes)
-ERROR: roles/add_day2_node/tasks/main.yml:44: var-naming[no-role-prefix]: Variables names from within roles should use patch_host_config_ as a prefix. (vars: patch_cluster_id)
-ERROR: roles/add_day2_node/tasks/main.yml:49: name[template]: Jinja templates should only be at the end of 'name'
-ERROR: roles/add_day2_node/tasks/main.yml:54: yaml[truthy]: Truthy value should be one of
-ERROR: roles/add_day2_node/tasks/main.yml:69: yaml[truthy]: Truthy value should be one of
-ERROR: roles/add_day2_node/tasks/main.yml:78: var-naming[no-role-prefix]: Variables names from within roles should use monitor_host_ as a prefix. (vars: current_host)
-ERROR: roles/add_day2_node/tasks/main.yml:78: var-naming[no-role-prefix]: Variables names from within roles should use monitor_host_ as a prefix. (vars: monitor_cluster_id)
-ERROR: roles/add_day2_node/tasks/main.yml:78: var-naming[no-role-prefix]: Variables names from within roles should use monitor_host_ as a prefix. (vars: waiting_termination_states)
```

It's really marking as errors the absence of the role prefix in the variables, even not changing them, but just because they're on the file I've modified.

Reply:
Gotcha, let's talk about this particular test to come up with an agreement

Reply:
That was done in an attempt to fix the ansible-lint check which requires the variables used in the role to be prefixed by the role's name. The attempt was unsuccessful because we also had these vars in the agents, and I didn't see it right away. 

After our yesterday discussion at daily, I think I'll do this prefix change in a separate PR.